modules/admin-ui-frontend/app/scripts/modules/registration/partials/terms-of-use.html

```diff
@@ -43,16 +43,14 @@ <h1 style="font-size: x-large;margin: 15px 0">
 </p>
 
 <p style="padding-bottom: 15px">
-  This data is anonymized and will be stored without any relation to your organizational
+  This data is anonymized and can be stored without any relation to your organizational
```

Comment:
*can* is quite vague. Could you specify in which case the usage statistics and technical data will be linked with the organizational data?

Reply:
Ah.  I meant that the data is anonymized, and can be stored w/o... *unless* you say we can link it internally.

What about something like `This data is anonymized by default, but will be linked to your organizational profile if you allow it.`?

Reply:
That sounds reasonable. Does that mean there is a checkbox (or something like it) in which admins can choose to link their organizational profile with the usage statistics and technical data?

Reply:
Yes.  `Which data do you want to share? > Usage statistics`, which is false by default.  So you can register your org, without sharing your hardware data.